components/interseller/actions/update-contact/update-contact.mjs

```diff
@@ -0,0 +1,37 @@
+import interseller from "../../interseller.app.mjs";
+
+export default {
+  key: "interseller-update-contact",
+  name: "Update Contact",
+  description: "Update an existing contact. [See the documentation](https://interseller.readme.io/reference/update-contact)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    interseller,
+    contactId: interseller.propDefinitions.contactId,
+    name: interseller.propDefinitions.name,
+    company: interseller.propDefinitions.company,
+    profileUrl: interseller.propDefinitions.profileUrl,
+    sourceUrl: interseller.propDefinitions.sourceUrl,
+    location: interseller.propDefinitions.location,
+    phoneNumber: interseller.propDefinitions.phoneNumber,
+    title: interseller.propDefinitions.title,
```

Comment:
Even though this works by copying the attributes directly within the code, I think we should adjust to follow the standard propDefinition syntax